app/services/game-overlay/index.ts

```diff
@@ -0,0 +1,396 @@
+import electron from 'electron';
+import { Subject, Subscription } from 'rxjs';
+import { delay, take } from 'rxjs/operators';
+import overlay, { OverlayThreadStatus } from '@streamlabs/game-overlay';
+import { Inject, InitAfter } from 'services/core';
+import { LoginLifecycle, UserService } from 'services/user';
+import { CustomizationService } from 'services/customization';
+import { getPlatformService } from '../platforms';
+import { WindowsService } from '../windows';
+import { PersistentStatefulService } from 'services/core/persistent-stateful-service';
+import { mutation } from 'services/core/stateful-service';
+import { $t } from 'services/i18n';
+import { Polly } from 'aws-sdk';
+
+const { BrowserWindow } = electron.remote;
+
+interface ICoordinates {
+  x: number;
+  y: number;
+}
```

Comment:
We have a ambient application type for this (you don't need to import it) called `IVec2` since it's such a common thing.